account_invoice_inter_company/models/account_move.py

```diff
@@ -79,6 +83,23 @@ def action_post(self):
             invoice.sudo()._set_intercompany_supplier_invoice_ref()
         return res
 
+    def _attach_original_pdf_report(self):
+        supplier_invoice = self.auto_invoice_id
+        if not supplier_invoice:
+            supplier_invoice = self.search([("auto_invoice_id", "=", self.id)], limit=1)
+        report = self.env.ref("account.account_invoices").with_company(self.company_id)
```

Comment:
```suggestion
        report = self.env.ref("account.account_invoices").with_company(supplier_invoice.company_id)
```

Reply:
Mmmh, we really want the customer invoice's pdf here (to attach it on the supplier invoice).
Why would we generate the pdf of customer invoice (which is `self`) using the supplier invoice's company ?
Maybe the `with_company` is kind of useless since the current company should already be `self.company_id`, but I don't understand why we would want to use the supplier one.

Reply:
OK, right. I have mixed things, but I think this would help if `self` is the supplier invoice, as it's the target of the action to do (attach there the customer invoice), and it's also advised to add docstrings. Let's not block this anyway.